common-rendering/src/components/keyEvents.tsx

```diff
@@ -82,6 +83,10 @@ const listStyles = (
 	supportsDarkMode: boolean,
 ): SerializedStyles => css`
 	${darkModeCss(supportsDarkMode)`
+		${until.desktop} {
+			background-color: ${neutral[10]};
```

Comment:
Would it be possible to capture this colour in `editorialPalette`?

Reply:
Do you mean creating a new function where we also pass the breakpoint as an argument?

Reply:
Got you. 
A hypothetical `keyEventsWide` would be reusing the same background as the article.

For clarity, do you think something like this would be helpful?
```typescript
const keyEvents = (_format: ArticleFormat): Colour => neutral[10];

const keyEventsWide = articleContentDark;
```
That way, the styles in the component would look like the code you wrote in the suggestion above.
Having a function pointing at another function is kind of redundant but would make the component code more readable imho.